packages/modelviewer.dev/data/examples.json

```diff
@@ -13,15 +13,15 @@
       },
       {
         "htmlId": "demo-container-3",
-        "name": "Customize Loading"
+        "name": "Customize Load"
       },
       {
         "htmlId": "demo-container-4",
-        "name": "Poster with invalid model"
+        "name": "Invalid Model"
```

Comment:
If we're on the documentation side, then `documentation` should not be clickable (or turn gray when hovering); likewise for examples.

Reply:
done